src/chocolatey/infrastructure.app/commands/ChocolateyFeatureCommand.cs

```diff
@@ -69,6 +64,19 @@ public virtual void ParseAdditionalArguments(IList<string> unparsedArguments, Ch
             }
 
             configuration.FeatureCommand.Command = command;
+
+            if ((configuration.FeatureCommand.Command == FeatureCommandType.List
+                 || !string.IsNullOrWhiteSpace(configuration.FeatureCommand.Name)
+                )
+                && unparsedArguments.Count > 1)
+            {
+                throw new ApplicationException("A single features command must be listed. Please see the help menu for those commands");

```

Comment:
This isn't a bad suggestion, but the verbiage used here is consistent with all the current existing commands.  If we decide we want to do what you are suggesting here (and I think it is a good idea) it would be a wholesale change across all the code.

Can you raise an issue for doing this?

Reply:
Sounds good. I've opened https://github.com/chocolatey/choco/issues/3121 for this.